yarn.lock

```diff
@@ -168,9 +168,9 @@
   dependencies:
     tslib "^1.7.1"
 
-"@ccd/case-ui-toolkit@0.0.29":
-  version "0.0.29"
-  resolved "https://artifactory.reform.hmcts.net:443/artifactory/api/npm/npm/@ccd/case-ui-toolkit/-/@ccd/case-ui-toolkit-0.0.29.tgz#f71303f8fef2fcd0cef5e4784d5c37290652f598"
+"@hmcts/ccd-case-ui-toolkit@1.0.2":
```

Comment:
@vlaurin, I think the latest version is 1.0.3 now https://www.npmjs.com/package/@hmcts/ccd-case-ui-toolkit

Reply:
not in master, according to the diff

Reply:
whoever has updated the version in their branch will update `yarn.lock`